components/climate/midea.rst

```diff
@@ -83,7 +82,6 @@ Configuration variables:
 - **uart_id** (*Optional*, :ref:`config-id`): Manually specify the ID of the :doc:`../uart` if you want
   to use multiple UART buses.
 - **name** (**Required**, string): The name of the climate device.
-- **transmitter_id** (*Optional*, :ref:`config-id`): Set if you use :doc:`../remote_transmitter` component for IR commands transmit.
```

Comment:
This should not be removed from the documentation. Documentation should have all config options.